doc/examples.xml

```diff
@@ -743,3 +746,26 @@ true
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="BinaryTree">
+<ManSection>
+  <Oper Name="BinaryTree" Arg="[filt, ]m"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    This function returns a binary tree of depth <A>m</A> with <C>2 ^ <A>m</A>
+    - 1</C> vertices. All edges are directed towards the root of the tree.
+    <P/>
+
```

Comment:
Do you think this is worth adding? It makes it a bit more explicit about which vertices are which in the tree.
```suggestion
    Note that <C>BinaryTree(<A>m</A>)</C> is the induced subdigraph of
    <C>BinaryTree(<A>m</A>+1)</C> on the vertices <C>[1..2^(<A>m</A>-1)]</C>.
    <P/>
    
```